lua/dracula/groups.lua

```diff
@@ -198,8 +199,8 @@ local function setup(configs)
       htmlLink = { fg = colors.purple, underline = true, },
       htmlSpecialChar = { fg = colors.yellow, },
       htmlSpecialTagName = { fg = colors.cyan, },
-      htmlTag = { fg = colors.pink, },
-      htmlTagN = { fg = colors.pink, },
+      htmlTag = { fg = colors.cyan, },
```

Comment:
[According to the Dracula Syntax Highlighting Specification](https://spec.draculatheme.com/#HtmlTags) I believe this should stay `pink`.

Reply:
Not saying you shouldn't look at that, but this repo doesn't conform to that at all lol. For example, the spec says functions should be green, but they're defined as yellow here.

Reply:
(I personally find the spec relies on pink for way too much, but I've also redefined nearly every group and mostly use this as a way to predefine the colors.)

Reply:
Functions are green in the Treesitter groups here, but it seems there is an inconsistency with the standard groups. I'd assume the goal of this project would be to conform to the spec as closely as possible while still allowing overrides in the `setup`.

Reply:
@mgerb ok :100: make a new PR